force_wfmanager/tests/test_click_run.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+from click.testing import CliRunner
+
+from force_wfmanager.run import force_wfmanager
+
+
+class TestClickRun(unittest.TestCase):
+    def test_click_main(self):
+        """Test if the command force_wfmanager works"""
```

Comment:
On gotcha here: unittest will print out the docstring of a test as the name of the test to output if one is present, eg. something like
```
Test if the command force_wfmanager works ... passed
```
instead of
```
test_click_main (TestClickRunner) ... passed
```
Which can make finding which function went wrong hard.  So normally I avoid docstrings in test methods and use a plain old comment if needed.